cmd/bosun/expr/funcs.go

```diff
@@ -344,6 +345,14 @@ func DropNA(e *State, T miniprofiler.Timer, series *Results) (*Results, error) {
 	}
 	return DropValues(e, T, series, 0, dropFunction)
 }
+func printGraphiteError(err error) {
```

Comment:
what do you mean, graphite errors are being produced in a variety of places.
And most of those locations are in the graphite library so it wouldn't be good to print the error there.
where do you suggest i inline this and print errors in a central place?